include/image-commands.mk

```diff
@@ -229,6 +229,19 @@ define Build/combined-image
 	@mv $@.new $@
 endef
 
+define Build/combined-ext-image
+	-sh $(TOPDIR)/scripts/om-fwupgradecfg-gen.sh \
```

Comment:
Do not silence the return code. I guess we would like to know if something went wrong. Remove the minus!

Does it work as well if you don't specific the to be used shell? Just the script path and done.